waku/v2/peermanager/discovery_connector.go

```diff
@@ -225,6 +225,7 @@ func (c *PeerConnectionStrategy) workPublisher(ctx context.Context) {
 				case <-ctx.Done():
 					return
 				case p := <-c.peerCh:
+					//TODO: Modify to use peermanager AddPeer
```

Comment:
Should we create an issue for this TODO?

Reply:
This will be taken up during another peer-manager task itself, hence not required.
Just did not want to club it with service slots.